components/calendar/src/calendar/navigation-container.js

```diff
@@ -0,0 +1,177 @@
+                    </div>
+                    <div className="curr">
+                        <span className="label">{currYear.label}</span>
+                    </div>
+                    <div className="next">
+                        <button
+                            onClick={nextYear.navigateTo}
+                            name="next-year"
+                            aria-label="Go to next year"
+                            type="button"
+                        >
+                            <NextIcon color={styleOptions.chevronColor} />
+                        </button>
+                    </div>
+                </div>
+            </div>
+            <style jsx>{`
+                button {
+                    background: none;
+                    border: 0;
+                }
+                .month,
+                .year {
+                    display: flex;
+                    width: 100%;
+                    align-items: center;
+                    justify-content: center;
+                }
+
+                .month .curr,
+                .year .curr {
+                    flex: 2;
+                    white-space: nowrap;
+                }
+
+                .prev,
+                .next {
+                    flex: 1;
+                    text-align: center;
+                }
+
+                .prev button,
+                .next button {
+                    padding: ${spacers.dp4};
+                }
+
+                .prev:hover button,
+                .next:hover button {
+                    background-color: ${colors.grey200};
+                    cursor: pointer;
+                }
+
+                .label {
+                    display: flex;
+                    padding: 4px 8px;
+                    justify-content: center;
+                    min-height: 16px;
+                }
+
+                .navigationContainer {
+                    height: ${spacers.dp36};
+                    gap: ${spacers.dp8};
+                    padding: ${spacers.dp4};
+                    display: flex;
+                    flex-direction: row;
+                    border-bottom: 1px solid ${styleOptions.wrapperBorderColor};
+                    background-color: ${styleOptions.headerBackground};
+                    font-size: 1.08em; /*15px*/
+                }
+                .navigationContainer > div {
+                    display: flex;
+                    align-items: flex-start;
+                }
+                .navigationContainer > .prev {
+                    justify-content: flex-start;
+                }
+                .navigationContainer > .curr {
+                    justify-content: center;
+                }
+                .navigationContainer > .next {
+                    justify-content: flex-end;
+                }
+                .navigationContainer > .today {
+                    grid-area: today;
+                }
+                .today > button {
```

Comment:
I don't see a today class at all? (I'm not sure if this is captured by Hendrik's previous comment or is this additional)